opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py

```diff
@@ -692,7 +692,7 @@ def get_logger(
         attributes: Optional[Attributes] = None,
     ) -> Logger:
         if self._disabled:
-            _logger.warning("SDK is disabled.")
+            warnings.warn("SDK is disabled.")
```

Comment:
+ 1 to this. Wdyt about using `warnings` instead of logger for other places in which are common to have recursions?

Reply:
+1 to this ⬆️

Reply:
We are already using warnings https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-sdk/src/opentelemetry/sdk/_logs/_internal/__init__.py#L209 in other places

Reply:
Will this break again if someone is using logging.captureWarnings()? https://docs.python.org/3/library/logging.html#integration-with-the-warnings-module

Reply:
> Wdyt about using warnings instead of logger for other places in which are common to have recursions?

We should open an issue and add it to the logging board :)

> Will this break again if someone is using `logging.captureWarnings()`? https://docs.python.org/3/library/logging.html#integration-with-the-warnings-module

I guess so but at least we don't ship code using that ourself :)

Reply:
That's true but it seems like a common configuration that will break our SDK. My point is, I don't think warnings alone will solve recursion issues

I'm OK to go ahead with this PR for the patch release.